plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TableSnapshot.java

```diff
@@ -200,7 +203,7 @@ public Stream<DeltaLakeTransactionLogEntry> getCheckpointTransactionLogEntries(
         return getCheckpointPartPaths(checkpoint).stream()
```

Comment:
https://github.com/delta-io/delta/blob/master/PROTOCOL.md#table-features

> A table must not use [multi-part checkpoints](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#multi-part-checkpoint)

Where are we checking this constraint?